persistence/data_models_json_fields.go

```diff
@@ -139,15 +281,16 @@ type WorkerTaskBackoffInfoJson struct {
 }
 
 type LocalQueueMessageInfoJson struct {
-	QueueName string    `json:"queueName"`
-	DedupId   uuid.UUID `json:"dedupId"`
+	QueueName string               `json:"queueName"`
+	DedupId   uuid.UUID            `json:"dedupId"`
+	Payload   xdbapi.EncodedObject `json:"payload"`
```

Comment:
We probably should remove this payload here. The payload should only live in the local_queue table. Otherwise the process execution will grow too big and cause perf issue to the database. When a state execution succesfully consume the message, we will:
1. select the payload from the table
2. move the payload into the commandResults
3. delete the row from the table.

Reply:
If deleting the row from local queue table, how could we check the duplicated dedupId?